R/helpers_simulations.R

```diff
@@ -0,0 +1,33 @@
+#' Convenience function to make barplots of percentages
+#'
+#' @param x vector of samples
+#' @param description xlab string
+#' @param xaxisround rounding for xaxis labels (default: 0, i.e. integers will
+#' be used)
+#'
+#' @return the ggplot2 object
+#'
+#' @keywords internal
+#' @importFrom ggplot2 ggplot geom_histogram aes xlab ylab xlim
+#' @example examples/barplot_percentages.R
```

Comment:
because it is internal, we should not have examples (they will fail because not exported) and we should also consider prefixing the name with `h_`